.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

suggestion: keep NPM packages up to date. I guess dependabot won't bother raising PRs for minor versions, as `^` will get minor updates (subject to yarn.lock)

```suggestion
version: 2
updates:

  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "monthly"
  - package-ecosystem: 'npm'
    directory: '/'
    schedule:
      interval: 'monthly'
```
husky & linting related dependencies have some major version updates. I checked for updates with `npm i -g npm-check-updates && ncu -u && npm i`.